node/packages/aws-lambda-otel-extension/external/otel-extension-external/index.js

```diff
@@ -280,7 +286,11 @@ module.exports = (async () => {
               case 'telemetryData':
                 lastTelemetryData = data;
                 if (data.record.responseEventPayload) {
-                  sendReport('response', stripResponseBlobData(data.record.responseEventPayload));
+                  const { eventData: currentRequestData } = getCurrentRequestData();
+                  sendReport(
+                    'response',
+                    createResponsePayload(data.record.responseEventPayload, currentRequestData)
```

Comment:
I'd stick to existing variable naming, and not name _eventData_ as _currentRequestData_, as in other parts `currentRequestData` is a different object.

Also I don't think it's necessary to create a variable here.

--- 

Notheless if some variables are not named right, we definitely should improve that, but I'd ensure we're using same naming convention in all parts

Reply:
@medikoo Alright I changed the name of that function `getCurrentRequestData` to `getCurrentRequestContext` and changed the `eventData` attribute to `requestData`.

I thought it would make more sense to call this `getCurrentRequestContext` since we are doing more than just storing request data in that object. Similarly I thought changing `eventData` to `requestData` would make that payload a bit more specific.

Let me know what you think 🙏